app/helpers/exams_helper.rb

```diff
@@ -0,0 +1,18 @@
+module ExamsHelper
+  def exam_information_for(user, exam)
+    %Q{
+      #{if user.teacher_here?
+          "#{fa_icon('graduation-cap', class: 'fa-fw',
+                     text: "<strong>#{t :course}:</strong> #{exam.course.canonical_code}".html_safe)}
+          <br>"
+        end
+      }
```

Comment:
Could you estract this logic to another method?

Reply:
Sure!